### PR TITLE
Options will be passed as ENV_variables from command line

### PR DESCRIPTION
**Pull request by mohanprasaths**
Fixes: [613](https://github.com/loadimpact/k6/issues/613)
The options sent from cli flags will also be saved as environment variables .

So they can be easily accessed as __ENV.XXXX

eg:
**Running**
`./k6 run  --vus 1 --iterations 1  --blacklist-ip 192.168.1.2/8,192.178.12.1/16 --duration 1s --summary-trend-stats "avg,p(95)" --system-tags check,group --tag test=p,l=0 --stage 5s:10,5m:20,10s:5  /media/sf_MCMP/Code/src/github.com/loadimpact/k6/test.js
`

**test.js**

```
export default function() {
    console.log("---START---")
    console.log("VUS" + __ENV.vus)
    console.log("duration" + __ENV.duration)
    console.log("stages" + __ENV.stages)
    console.log("throw" + __ENV.throw)
    console.log("systemTags" + __ENV.systemTags)
    console.log("blackListIPS" + __ENV.blacklistIPs)
    console.log("tags" + __ENV.tags)
    console.log("---END---")
}
```

**Output:**

> INFO[0002] Setup done                                   
> INFO[0004] ---START---                                  
> INFO[0004] VUS1                                         
> INFO[0004] duration1s                                   
> INFO[0004] stages[{"duration":"5s","target":10},{"duration":"5m0s","target":20},{"duration":"10s","target":5}] 
> INFO[0004] throwfalse                                   
> INFO[0004] systemTags{"check":true,"group":true}        
> INFO[0004] blackListIPS["192.0.0.0/8","192.178.0.0/16"] 
> INFO[0004] tags{"l":"0","test":"p"}                     
> INFO[0004] ---END---                                    
> INFO[0004] tear down done

### CODE REVIEW

**Comment by mohanprasaths** [comment]
Thanks for helping @na-- . 
I have doubt here. 
I understand the options variable has to be exported back to scripts but how they should be exposed in scripts. ie: how the real values can be read .
is that via `module.exports` or
 any new export key like  `rt.Set("__OPTIONS", OPTIONS)
`
or _any other better way_

**Comment by mohanprasaths** [comment]
@na--  I have updated my commit , Please check it

File: lib/options.go [hunk 1]
```diff
@@ -186,80 +186,80 @@ type Options struct {
 
 	// Initial values for VUs, max VUs, duration cap, iteration cap, and stages.
 	// See the Runner or Executor interfaces for more information.
-	VUs        null.Int           `json:"vus" envconfig:"vus"`
-	VUsMax     null.Int           `json:"vusMax" envconfig:"vus_max"`
-	Duration   types.NullDuration `json:"duration" envconfig:"duration"`
-	Iterations null.Int           `json:"iterations" envconfig:"iterations"`
-	Stages     []Stage            `json:"stages" envconfig:"stages"`
+	VUs        null.Int           `json:"vus" envconfig:"vus" js:"vus"`
```
Comment (na--) [inline lib/options.go]: What are the `js:` struct tags used for?

Comment (mohanprasaths) [reply lib/options.go]: [here](https://github.com/loadimpact/k6/blob/master/js/common/bridge.go#L47)
when a struct is exported in to goja runtime, the property names are converted to snake_cased if "js" tag is missing in the struct
ie: VUs -> v_us , VUsMax -> v_us_max

Comment (na--) [reply lib/options.go]: Ah, I see, makes total sense.

**Comment by na--** [comment]
Thanks for the test and for squashing the code! I'm merging this and later I'll probably expand the test slightly to also verify that the JS code sees the new options as well

**Pull request merged**